### PR TITLE
Added Q/A on faq regarding downloading & setting up a vagrant box manually #491

### PR DESCRIPTION
**Pull request by ztickm**
Answering #491 
Recap:
> If someone wants to run BeLL for the first time, they will have to run vagrant up which will try to download a big file which is a vagrant box. With no pause/resume download functionality, an internet interruption will force the user to download the file again from the beginning which will discourage people with slower internet connections.

I added a Q/A on FAQ guiding users on how to manually download and set up a vagrant box directly from [ole's jessie64 atlas box page](https://atlas.hashicorp.com/ole/boxes/jessie64/). I added two images (atlas_last_version_box.png and atlas_status.png) to accompany the answer, they are in `uploads/images/` 

P.S. 	 commits [`f3514ee`](https://github.com/ztickm/ztickm.github.io/commit/f3514ee35563d252ee8a92a0a373ead5a4368c90) and above are just fork syncs they should not affect the repo if merged, the real modifications are in the last commit [`4be1760`](https://github.com/ztickm/ztickm.github.io/commit/4be1760b3446350072b998ff8a97f6f38533f152). If something needs to be fixed just tell me, I'll be happy to get your comments and directions.

### CODE REVIEW

**Comment by dogi** [comment]
@ztickm interesting

@dogi has to check if that also really works ;)

Anyhow do not agree with the place where you put it in ...

File: pages/faq.md [hunk 1]
```diff
@@ -29,6 +29,22 @@ To finish the initial steps, you need to make a **minimum** of three issues and
 There is no official deadline, work on your own time. However, please note that most people who continued into the internship program completed the steps within 7-8 days. 
 ## Technical Issues and Questions
 
+*Q: When I first run BeLL with the `vagrant up` command the download fails, why?*
```
Comment (dogi) [inline pages/faq.md]: @ztickm please move this to a later position ...

File: pages/faq.md [hunk 2]
```diff
@@ -41,10 +40,32 @@ Please make sure you’re using Firefox, as our BeLL software is only guaranteed
...
+You should first check if [Hashicorp's atlas](https://atlas.hashicorp.com/boxes/search) is up and running by looking at the left sidebar :
+
+![left sidebar](uploads/images/atlas_status.png)
+
+If Atlas is operational, then maybe your download is being interrupted, you can try to download and set up the big vagrant box file manually:
+
+1. Go to this [atlas box page](https://atlas.hashicorp.com/ole/boxes/jessie64/)
+
+2. Click on the last version's (the uppermost) version number
```
Comment (hikaruit15) [inline pages/faq.md]: Please fix that
screen shot 2017-02-15 at 1 13 30 pm

**Comment by ztickm** [comment]
Corrected

**Comment by hikaruit15** [comment]
Hi @ztickm,
When you fix it, the order also changes.
I hope you can do the order correctly :)
pr_493

(P/s: Using [Gitter chat](https://gitter.im/open-learning-exchange/chat) to inform what you do)

**Comment by ztickm** [comment]
This is driving me nuts, I return lines before and after an image, it sits nicely where it should be, but it breaks the list. If I delete a line return the image sticks to a text.

**Comment by ztickm** [comment]
Plus the github markdown preview looks nothing like the actual result on MDWiki

**Comment by dogi** [comment]
@ztickm https://ztickm.github.io/#!pages/faq.md looks good

**Comment by ztickm** [comment]
Thanks @dogi, I'll look more into styling in MDwiki maybe I will be able to make it better.

**Comment by hikaruit15** [comment]
Hi @ztickm,
Now you are back to the first error that I mentioned.
Basically you are very close to solution.
Just keep the return line and add a space or a tab before image. It will work well.
solution_pr_493

**Comment by hikaruit15** [comment]
Hi @ztickm,
Have you check your page after changing your code? It seems nothing change.
Sorry if I made you are confused
>Just **keep the return line** and add a space or a tab before image.

I mean you will do a return line after this line `2. Click on the last version's (the uppermost) version number`
Then try a tab or space before image.
Please look the image at my previous comment.

**Pull request merged**